Updating Tutorials `[AARD-2091]` (#1286)
Autodesk/synthesis: tutorials/MatchMode.md

author: Synthesis Team
summary: Tutorial for starting a simulated match.
id: MatchMode
tags: Match, Gameplay, Simulate
categories: Gameplay
environments: Synthesis
status: Draft
feedback link: https://github.com/Autodesk/synthesis/issues

## Introduction

> Match mode in Synthesis simulates competitive robotics matches with autonomous and teleoperated periods, mimicking real competition environments. It also keeps track of individual robot scores, and penalties, giving a breakdown at the end. 

Reviewer: ```suggestion
Match mode in Synthesis simulates competitive robotics matches with autonomous and teleoperated periods, mimicking real competition environments. It also keeps track of individual robot scores and penalties, giving a breakdown at the end. 
```

## Starting a Match

To get started with running a match, first spawn and configure the field and robot(s) you want to use as you normally would (link to spawning robot tutorial). 

> Next open up the MainHUD and press start match mode.

Reviewer: ```suggestion
Next, open up the MainHUD and press start match mode.
```

This will open up the Match Mode Config panel, where you can configure the match mode parameters. 

<img src="img/matchmode/match-mode-panel.png" alt="View of panel to select which year's match mode configuration you would like to use" width="300">

> There are already multiple preconfigured match modes to use, and if you are simulating a match for that corresponding year, you can just select a preconfigured ruleset. This will start the match. You can always stop the match by opening the MainHUD and pressing the abort match mode button that appears in place of start match. 

Reviewer: ```suggestion
There are multiple pre-configured match modes. If you’re simulating a match for a specific year, simply select the corresponding ruleset. This will start the match. You can always stop the match by opening the MainHUD and pressing the abort match mode button that appears in place of start match. 
```
